api/Document.json

```diff
@@ -2049,13 +2049,16 @@
           "spec_url": "https://dom.spec.whatwg.org/#ref-for-dom-document-createelement①",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Chromium still returns an Element."
```

Comment:
Setting aside the correctness discussion on the content PR, this note isn't descriptive enough to stand alone (on caniuse and other non-MDN contexts). "Still" with respect to when or what? If Chrome etc. are doing something non-standard or otherwise surprising, the notes should spell that out.

Additionally, avoid referencing browser engines in notes and use the actual browser's name instead (i.e., Chrome not Chromium, Edge not Chromium, and so on).